masonry/src/widget/sized_box.rs

```diff
@@ -371,7 +365,13 @@ impl Widget for SizedBox {
 
             trace_span!("paint background").in_scope(|| {
                 scene.push_layer(BlendMode::default(), 1., Affine::IDENTITY, &panel);
-                background.paint(ctx, scene);
+                scene.fill(
+                    Fill::NonZero,
+                    Affine::IDENTITY,
+                    &*background,
+                    Some(Affine::IDENTITY),
+                    &ctx.size().to_rect(),
+                );
                 scene.pop_layer();
```

Comment:
```suggestion
                scene.fill(
                    Fill::NonZero,
                    Affine::IDENTITY,
                    &*background,
                    Some(Affine::IDENTITY),
                    &panel,
                );
```

Reply:
So, we don't need a new layer for the background now?

Reply:
Yeah - the layer was previously needed because of the paint function, but because it's just a single item, this works as expected.